src/validation.cpp

```diff
@@ -1524,6 +1524,7 @@ static uint32_t GetNextBlockScriptFlags(const Consensus::Params &params,
     // Start enforcing P2SH (BIP16)
     if ((pindex->nHeight + 1) >= params.BIP16Height) {
```

Comment:
You probably know, but these old softfork checks will need to go too. Did you try cleanstack outside of this?